src/components/BlockStats.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <b-card v-if="graphData" title="Transactions Per Block">
```

Comment:
I suggest using a title like "Transactions Processed" to indicate that only blocks produced by this node count.